jira-workflows/goose/check-jira-tickets/recipe.yaml

```diff
@@ -0,0 +1,20 @@
+version: 0.1.0
+title: "Check if there is any Jira issue we could be interested in"
+description: "Check issues for {{ component }} in {{ project }}"
+
+
+instructions: |
+  - Never create, delete, update or modify an Issue in Jira
```

Comment:
We need to make sure that the Jira token we pass through has limited privileges.  The agent should only be able to read and comment.

Having those explicit instructions here is important to set the guardrails and prevent the agent from trying.  But the token permissions will make sure we prevent accidents.

Reply:
> If you agree I can open an issue for it?

That would be great, thanks!

So we may need to set up a new account for a Jira bot with those limited permissions.  I will ask Rui on Slack.